ci-operator/step-registry/cucushift/hypershift-extended/OWNERS

```diff
@@ -1,6 +1,8 @@
 approvers:
-  - LiangquanLi930
-  - heliubj18
+- LiangquanLi930
+- heliubj18
+- fxierh
```

Comment:
Based on the contributions, I don't think now is an appropriate time, So don't change it

Reply:
1. You do not have the authority to make this decision.
2. You are not the one to evaluate other people's contributions.
3. This should not be subject to your personal biases.

Reply:
I will raise a separate PR to give permission to fxierh in hypershift-extendend folder.